api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityClient.java

```diff
@@ -29,7 +30,7 @@
     private final String eligibilityUri;
 
     public EligibilityClient(@Value("${eligibility.base-uri}") String baseUri,
-                             RestTemplate restTemplateWithIdHeaders,
+                             @Qualifier("restTemplateWithIdHeaders") RestTemplate restTemplateWithIdHeaders,
```

Comment:
Now we have two rest templates we need this anywhere we use a rest template.

Reply:
In which case we need to be using a constant for the qualifier - hopefully there is one in common-rest?